pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -331,8 +334,36 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
-	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
+func getIpsFromResult(res *cnicurrent.Result, ifaceName string) []net.IP {
+	if res == nil {
+		klog.V(3).Info("res is nil")
+		return nil
+	}
+	ipv6DualStack := utilfeature.DefaultFeatureGate.Enabled(kubefeatures.IPv6DualStack)
```

Comment:
IMHO the check if `IPv6DualStack` is enabled is not required. We could  remove the `if ipv6DualStack` check and just return always the `outlist` as a result (for non `IPv6DualStack` cluster with one item otherwise with two).

Reply:
done